docs/content/faq.md

```diff
@@ -85,10 +85,11 @@ docker run --rm -it \
 find /backup/mail -type f -mtime +30 -delete
 ```
 
-### What about `mail-state` folder?
+### What is the `mail-state` folder for?
 
-This folder consolidates all data generated by the server itself to persist when you upgrade.
-Example of data folder persisted: lib-amavis, lib-clamav, lib-fail2ban, lib-postfix, lib-postgrey, lib-spamassasin, lib-spamassassin, spool-postfix, ...
+When you run `docker-mailserver` with the ENV var `ONE_DIR=1` (_default since v10.2_), this folder will store the data from internal services so that you can more easily persist state.
```

Comment:
How about this?

```suggestion
When you run `docker-mailserver` with the ENV var `ONE_DIR=1` (_default since v10.2_), this folder will store the data from internal services so that you can more easily persist state to disk (via `volumes`).

This has the advantage of fail2ban blocks, ClamAV anti-virus updates and the like being kept across restarts for example.
```